src/components/form/WebTechnologiesInput.svelte

```diff
@@ -6,20 +6,59 @@
   bind:value
 />
 
-<AddOther
-  label="Add other {TECHNOLOGY_NAME_LABEL}"
-  on:ADD="{handleTechnologyAdd}"
->
-  <Input
-    id="technologies_relied_upon__other--name"
-    label="{TECHNOLOGY_NAME_LABEL}"
-    helptext="{TECHNOLOGY_HELPTEXT}"
-  />
-  <Input
-    id="technologies_relied_upon__other--url"
-    label="{TECHNOLOGY_URL_LABEL}"
-  />
-</AddOther>
+<div class="Editable">
+  <legend>{BTN_ADD_TECH}</legend>
+  <AddOther
+    label="{ADD_TECH}"
+    on:ADD="{handleTechnologyAdd}">
+    <div class="tech-input">
+      <Input
+        id="technologies_relied_upon__other--name"
+        label="{TECHNOLOGY_NAME_LABEL}"
+      />
+      <Input
+        id="technologies_relied_upon__other--url"
+        label="{TECHNOLOGY_URL_LABEL}"
+      />
+    </div>
+  </AddOther>
+</div>
+
+<style>
+  .Editable__Control--delete {
+    font-size: 1em;;
+    float: right;
+    margin-left: 1em;
+    margin-top: .25em;
+    padding: 0;
+    border: 0;
+    background-color: transparent;
+    color: currentColor;
+  }
+  .Editable {
```

Comment:
We need the styles in this rule, but the name is confusing I think. Mabe instead of the classname`Editable` we can use something like `add-web-technology`?